packages/percy-storybook/src/getStaticAssets.js

```diff
@@ -68,15 +68,19 @@ function gatherBuildResources(buildDir) {
   return hashToResource;
 }
 
+export function getStorybookJavascriptPath(storyHtml) {
+  return storyHtml.match(
+    /<script (?:type="text\/javascript" )?src="(.*?static\/preview.*?)"><\/script>/,
```

Comment:
More flexible:
```regex
/<script [^>]*src\s*=\s*["'](.*?static\/preview.*?)["'][^>]*>/,
```

And it works:
![screen shot 2017-10-06 at 9 51 59 am](https://user-images.githubusercontent.com/75300/31289008-24877254-aa7c-11e7-8f91-44df6da3cd93.png)
https://regex101.com/r/DmqgIu/1